ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -813,15 +823,15 @@ port_channel_interfaces:
   < Port-Channel_interface_2 >:
     description: < description >
     vlans: "< list of vlans as string >"
-    mode: < access | trunk >
+    mode: < access | dot1q-tunnel | trunk | trunk-phone >
     esi: < EVPN Ethernet Segment Identifier (Type 1 format) >
     rt: < EVPN Route Target for ESI with format xx:xx:xx:xx:xx:xx >
     lacp_id: < LACP ID with format xxxx.xxxx.xxxx >
   < Port-Channel_interface_3 >:
     description: < description >
     vlans: "< list of vlans as string >"
     type: < routed | switched | l3dot1q >
-    mode: < access | dot1q-tunnel | trunk >
+    mode: < access | dot1q-tunnel | trunk | trunk-phone >
```

Comment:
This should be `< access | dot1q-tunnel | trunk | "trunk phone" >`

Reply:
Good catch.  This should be resolved.